<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatDisplay</class>
 <widget class="QWidget" name="StatDisplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>120</width>
    <height>120</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>120</width>
    <height>120</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>120</width>
    <height>120</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="statNumber">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="statModifier">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>65</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="statName">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>STR</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
